node/actors/bft/src/leader/replica_commit.rs

```diff
@@ -140,12 +151,21 @@ impl StateMachine {
             by_proposal.entry(msg.msg.proposal).or_default().push(msg);
         }
         let threshold = self.config.genesis().validators.threshold();
-        let Some((_, replica_messages)) =
-            by_proposal.into_iter().find(|(_, v)| v.len() >= threshold)
+        let Some((proposal, _replica_messages)) = by_proposal
```

Comment:
nit: there is a semantic difference between nameing a message "_" (value is dropped immediately) and "_sth" (value is dropped at the end of the scope).